templates/thresh-config.yml.j2

```diff
@@ -1,6 +1,13 @@
 metricSpoutThreads: 2
 metricSpoutTasks: 2
 
+statsdConfig:
+  host: {{mini_mon_host}}
+  port: 8125
+  prefix: monasca.storm.metrics.
+  dimensions: '{"hostname":"localhost","service":"monitoring","component":"storm"}'
```

Comment:
The agent will by default set the hostname, it doesn't need to be specified here. Having the agent specify it is advantageous because it will set it to what the machine actually is at.